### ISSUE TITLE
incompatibility with pydocstyle==6.2.0

### ISSUE THREAD

**Issue by jvesely** [henry0312/pytest-pydocstyle#91]
```
________________________ ERROR collecting test session ________________________
d:\a\_temp\_venv\lib\site-packages\pluggy\_hooks.py:265: in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
d:\a\_temp\_venv\lib\site-packages\pluggy\_manager.py:80: in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
d:\a\_temp\_venv\lib\site-packages\pytest_pydocstyle.py:50: in pytest_collect_file
    for filename, _, _ in parser.get_files_to_check():
E   ValueError: too many values to unpack (expected 3)
```

**Comment by henry0312** [comment]
Thank you for report. I'll check.

**Comment by jvesely** [comment]
still present in pydocstyle==6.2.2

**Comment by odoublewen** [comment]
For cross-ref, I opened this issue over in pydocstyle yesterday:
https://github.com/PyCQA/pydocstyle/issues/620
Someone posted a clue about the cause.

**Comment by henry0312** [comment]
This issue should be solved since [v2.3.2](https://github.com/henry0312/pytest-pydocstyle/releases/tag/v2.3.2).
Thank you!
